docs/DeveloperGuide.md

```diff
@@ -23,12 +131,274 @@
 
 ## Non-Functional Requirements
 
-{Give non-functional requirements}
+### Performance and scalability
+
+* Constraint: Multi-User
+  
+The product allows user to choose to login to different accounts upon start up of software
+
+* Constraint: Typing-Preferred
+
+The software is targeted for user who prefers typing as means of input
+
+### Portability and compatibility
+
+* Constraint: Platform-Independent
+
+The software is runnable on the Windows, Linux, and OS-X platforms.
+
+* Constraint: Java-Version
+
+The software is written and tested in Java 11.
+
+### Reliability, availability, maintainability
+
+### Security
+
+ * Constraint: Non-editable Local Storage File
+
+The local storage file cannot be modified by user directly.
+
+### Usability
 
 ## Glossary
 
 * *glossary item* - Definition
 
 ## Instructions for manual testing
-
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+### Logging into system 
+
+#### 1. Login
+##### Test case 1.0 : When correct login information is provided
+Logs into the system when correct email address and password is provided by the user
+
+**Test case**:
+
+Enter choice: `1`
+
+Enter email address: `12312@gmail.com`
+
+Enter password: `5678`
+
+Expected:
+```` Hello! I'm MojoHR
```

Comment:
![Screenshot 2021-04-03 at 12 34 19 PM](https://user-images.githubusercontent.com/65282395/113468455-eb52f180-9478-11eb-8833-eea690a8dac1.png)

Should the format for code fencing be 4 "````" ? If changed to the correct 3 "```" The expected output will be much neater and standardised with the other outputs.